DepositeCalcTests/Tests/LoginPageTests.cs

```diff
@@ -7,12 +7,14 @@ namespace DepositeCalcTests.Tests
     public class LoginPageTests : BaseTest
     {
         private LoginPage loginPage;
+        private RemindPasswordView remindPasswordView;
 
         [SetUp]
         public void Setup()
         {
             InitDriver("https://localhost:5001/");
             loginPage = new LoginPage(driver);
+            remindPasswordView = new RemindPasswordView(driver);
```

Comment:
Please rollback this change

Reply:
@varajan I need your explanation about this rollback